### PR TITLE
Compute V2: Create multiple servers

### PR DESCRIPTION
**Pull request by code0x9**
For #1306 

API:
https://developer.openstack.org/api-ref/compute/?expanded=create-multiple-servers-detail#create-server

Schema:
https://github.com/openstack/nova/blob/5852bae034533129608dd24b58e33733e093e957/nova/api/openstack/compute/schemas/servers.py#L195

### CODE REVIEW

File: openstack/compute/v2/servers/requests.go [hunk 1]
```diff
@@ -183,9 +183,18 @@ type CreateOpts struct {
...
 	AccessIPv6 string `json:"accessIPv6,omitempty"`
 
+	// Min specifies Minimum number of servers to launch.
+	Min int `json:"min_count,omitempty"`
+
+	// Max specifies Maximum number of servers to launch.
+	Max int `json:"max_count,omitempty"`
+
+	// ReturnReservationID sets respond with reservation_id only.
+	ReturnReservationID bool `json:"return_reservation_id,omitempty"`
```
Comment (jtopjian) [inline openstack/compute/v2/servers/requests.go]: If I understand this correctly, setting this to `true` will cause the response body to be a different format? If so, then this will cause a JSON error when running `Create` and `Extract`. To get around this, we will need to create a new `Extract` method to handle this. This seems to be out of scope for the original issue this PR is fixing, so let's remove this.

Comment (code0x9) [reply openstack/compute/v2/servers/requests.go]: right. just removed that code. let's do it when it is needed.

**Comment by jtopjian** [comment]
@code0x9 This looks good to me. Let me know if you're done working on this and I'll merge it. 

The OpenLab test failure can be ignored.

**Comment by code0x9** [comment]
@jtopjian I'm done. please merge this.
